### DISCUSSION TITLE
How to use compare in keystone4?

### DISCUSSION THREAD

**Discussion by wendylw** [keystonejs/keystone#7867] [Questions] [answered]
in routes/views, how to use keystone password compare function to compare requesting body password with saved password?

**Comment by dcousens** [comment, marked as answer]
Hi @wendylw,

This is the `keystone-6` repository,  you can find keystone 4 at https://github.com/keystonejs/keystone-classic, but we (@keystonejs/maintainers) are not actively maintaining that project.
